src/Error.ts

```diff
@@ -18,7 +18,7 @@ type StripeRawError = {
   doc_url?: string;
   decline_code?: string;
   param?: string;
-  detail?: string;
+  detail?: any;
```

Comment:
To account for different detail type here: https://github.com/stripe/stripe-node/blob/84d3ec43b0dd582fee811f2330f2001095e90475/src/Webhooks.ts#L174

Reply:
Thanks for the suggestion! I went with the second option (upgraded to TypeScript 4+ because @ts-expect-error required TS 3.9+ and I forgot to upgrade previously)